changelogs/CHANGELOG.rst

```diff
@@ -5,6 +5,19 @@ Vyos Collection Release Notes
 .. contents:: Topics
 
 
+v1.0.4
+======
+
+Minor Changes
+-------------
+
+- Moved intent testcases from integration suite to unit tests.
+
+Bugfixes
+--------
+
+- Added error pattern to the terminal plugin to handle change mode error seen in lag interfaces config.
```

Comment:
for future changelogs - it helps if you add a link to either the issue or the PR that made the change.